python/ray/train/_internal/state/schema.py

```diff
@@ -1,10 +1,24 @@
+from enum import Enum
 from typing import List, Optional
 
 from ray._private.pydantic_compat import BaseModel, Field
 from ray.dashboard.modules.job.pydantic_models import JobDetails
 from ray.util.annotations import DeveloperAPI
 
 
+@DeveloperAPI
+class RunStatusEnum(str, Enum):
+    STARTED = "STARTED"
+    FINISHED = "FINISHED"
+    ERRORED = "ERRORED"
+
+
+@DeveloperAPI
+class ActorStatusEnum(str, Enum):
```

Comment:
is there an enum we can re-use for this?

Reply:
hmm I didn't find an Enum class for python, there are only some literal definitions([link](https://github.com/ray-project/ray/blob/05067f4955e6e79e3cbf05fe6875677de5663924/python/ray/util/state/custom_types.py#L17)) or Enum class for ts([link](https://github.com/ray-project/ray/blob/05067f4955e6e79e3cbf05fe6875677de5663924/dashboard/client/src/type/actor.ts#L3)).

Reply:
ok